src/patternfly/components/Expandable/examples/expandable-expanded-example.hbs

```diff
@@ -0,0 +1,7 @@
+{{#> expandable expandable--IsExpanded="true"}}
+  {{#> expandable-toggle expandable-toggle--attribute='aria-expanded="true"'}}
```

Comment:
aria-expanded should be false. however, could you just put a conditional in `expandable-toggle.hbs` that looks for `expandable--IsExpanded`, and generates the proper `aria-expanded` property so all you have to do is specify `expandable--IsExpanded` and everything else updates appropriately?